backend/app/resources/create_user_info_dto.py

```diff
@@ -0,0 +1,18 @@
+class CreateUserInfoDTO:
+    def __init__(self, **kwargs):
+        self.contact_name= kwargs.get("contact_name")
+        self.contact_email = kwargs.get("contact_email")
+        self.contact_phone = kwargs.get("contact_phone")
+        self.role = kwargs.get("role")
+
+    def validate(self):
+        error_list = []
+        if type(self.contact_name) is not str:
+            error_list.append("The contact name supplied is not a string.")
+        if type(self.contact_email) is not str:
+            error_list.append("The contact email supplied is not a string.")
+        if type(self.contact_phone) is not str:
+            error_list.append("The contact phone supplied is not a string.")
+        if type(self.role) is not str:
+            error_list.append("The role supplied is not a string.")
```

Comment:
Might be good to do additional validation on these fields, to ensure they match the expected format. For example, emails should match an email regex, phones should match a phone regex, roles should be in `["ASP", "Donor", "Admin"]`, etc.